### ISSUE TITLE
Cannot build on clean clone

### ISSUE THREAD

**Issue by barbeque-squared** [AnandChowdhary/language-icons#7]
How to reproduce:
* Clone
* Follow the instructions in the Building section of the readme (I did `yarn install --frozen-lockfile; npm run build`)

Expected result: it builds

Actual result:
```
[language-icons]$ npm run build

> language-icons@0.3.0 build /home/bbq/language-icons
> tsc && node dist/index.js

Generating 184 icons...
Error [Error: ENOENT: no such file or directory, open '/home/bbq/language-icons/icons/aa.svg'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/home/bbq/language-icons/icons/aa.svg'
}
```

I see two possible ways to fix this:
* the `files.ts#write` function should (try to) create the `icons` directory if it doesn't exist yet, or
* the Building instructions should simply mention it -- it's only a one-time operation like installing the dependencies anyway

**Comment by AnandChowdhary** [comment]
You're right, it's as easy as telling people to create the `icons` directory, and we can also ensure it exists before starting. Will be happy to accept a PR for either.
